match/lib/match/options.rb

```diff
@@ -72,10 +71,30 @@ def self.available_options
                                      code_gen_sensitive: true,
                                      default_value: CredentialsManager::AppfileConfig.try_fetch_value(:app_identifier),
                                      default_value_dynamic: true),
+
+        # App Store Connect API
+        FastlaneCore::ConfigItem.new(key: :api_key_path,
+                                     env_name: "SIGH_API_KEY_PATH",
+                                     description: "Path to your App Store Connect API Key JSON file (https://docs.fastlane.tools/app-store-connect-api/#using-fastlane-api-key-json-file)",
+                                     optional: true,
+                                     conflicting_options: [:username],
```

Comment:
Shouldn't this also include the `api_key`?
```suggestion
                                     conflicting_options: [:api_key, :username],
```

Reply:
Yes and no 🤷‍♂️  It's already taken care of by another conflicting option but should probably do this for clarify